src/mem/cache/tags/sector_tags.cc

```diff
@@ -274,7 +285,8 @@ SectorTags::findBlock(Addr addr, bool is_secure) const
 
 CacheBlk*
 SectorTags::findVictim(Addr addr, const bool is_secure, const std::size_t size,
-                       std::vector<CacheBlk*>& evict_blks)
+                       std::vector<CacheBlk*>& evict_blks,
+                       const uint64_t partition_id)
```

Comment:
Something must be done with the partition id here, as the implementation seems pretty trivial (preferred). Either that or assert that PPs are not used when this tag type is used.

Reply:
Same as CompressedTags, I've added an initial implementation of filtering, but as I'm not 100% confident in the expected behaviour of SectorTags when partitioned, I've added a warning for users in the constructor.